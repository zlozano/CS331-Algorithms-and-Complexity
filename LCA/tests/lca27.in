11 (12) (9)
10 () (9)
12 () (11)
1 (2,3) ()
3 () (1)
2 (4,5) (1)
5 (6,7) (2)
4 () (2)
7 (9) (5)
6 (8) (5)
9 (10,11) (7)
8 () (6)
