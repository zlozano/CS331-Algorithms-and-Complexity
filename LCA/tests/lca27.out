 1  1  1  1  1  1  1  1  1  1  1  1
 1  2  1  2  2  2  2  2  2  2  2  2
 1  1  3  1  1  1  1  1  1  1  1  1
 1  2  1  4  2  2  2  2  2  2  2  2
 1  2  1  2  5  5  5  5  5  5  5  5
 1  2  1  2  5  6  5  6  5  5  5  5
 1  2  1  2  5  5  7  5  7  7  7  7
 1  2  1  2  5  6  5  8  5  5  5  5
 1  2  1  2  5  5  7  5  9  9  9  9
 1  2  1  2  5  5  7  5  9 10  9  9
 1  2  1  2  5  5  7  5  9  9 11 11
 1  2  1  2  5  5  7  5  9  9 11 12
